libs/components/ag-grid/src/lib/modules/ag-grid/cell-validator/ag-grid-cell-validator-tooltip.component.ts

```diff
@@ -50,10 +52,30 @@ export class SkyAgGridCellValidatorTooltipComponent {
       }
     });
 
+    this.cellRendererParams.eGridCell?.addEventListener('mouseover', () => {
+      if (!this._hoverTimeout) {
+        this._hoverTimeout = setTimeout(() => {
```

Comment:
You can call `window.setTimeout`, here, to instruct the TypeScript compiler to reference the DOM's version of `setTimeout`, instead of importing the `Timeout` type from `NodeJS`. We probably want to use the DOM API since this component is used on a webpage?

Reply:
Also, it would be helpful to future maintainers if we provided a comment explaining why a timeout is needed, with a link to a StackOverflow/Github issue, if applicable.